libs/components/datetime/src/lib/modules/timepicker/fixtures/timepicker-input-box-component.fixture.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  <sky-input-box>
+  <sky-input-box labelText="Time">
```

Comment:
I know its just a test fixture but if you're using `labelText` then you don't need the label on the line below

Reply:
Fixed